core/tests/protractor_desktop/extensions.js

```diff
@@ -52,7 +52,6 @@ describe('rich-text components', function() {
       await richTextEditor.appendPlainText(' ');
       // TODO(Jacob): Add test for image RTE component.
       await richTextEditor.addRteComponent('Link', 'http://google.com/', true);
-      await richTextEditor.addRteComponent('Math', 'abc');
```

Comment:
Note: After discussion with @DubeySandeep and @kevintab95, it was decided to move the math RTE test to a test suite that does not run in production mode.
This is because, after this PR the Math RTE components will save an image in the backend, but currently in e2e tests running in prod-mode don't support GCS.

Reply:
I think we should also create a separate suite for fileUploadExtension which will cover all the extension which needs file upload. In future, we can add an imageRTE component test in the same suite. What do you think, this will also assure that we have a separate test for these extensions like we have for other RTE components. What do you think?

Reply:
Yeah makes sense, but that could be done in future PR right?

Reply:
@donosco98 creating the `fileUploadExtension` suite can be done in a separate PR, but the Math RTE tests should be in a running suite when this PR merges. I think creating a new suite for the Math RTE tests and creating the 1fileUploadExtension` suite will be about the same amount of work, but I think either way is fine

Reply:
@U8NWXD, I've added a test for math RTE in the playVoiceover.js file.
Should I change anything now?

Reply:
Also the complete tests for math RTE will be added once we start using the SVGs for displaying the math equations